src/Drawer/stories.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, select, text } from '@storybook/addon-knobs/react';
+
+import Drawer from './index';
+import Button from '../Button';
+import HxDiv from '../HxDiv';
+
+const SIZES = {
+  small: 'small',
+  medium: 'medium',
+  large: 'large',
+};
+
+storiesOf('Drawer', module).add('All Knobs', () => {
+  let header = text('header', '');
+  let body = text('body', '');
+  let footer = text('footer', '');
+  let open = boolean('open', true);
+  let size = select('size', SIZES, 'medium');
+
+  const loremIpsum = (
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+      labore et dolore magna aliqua. Gravida rutrum quisque non tellus. Sagittis vitae et leo duis
+      ut diam quam nulla. Diam vel quam elementum pulvinar etiam non. Pulvinar sapien et ligula
+      ullamcorper malesuada proin libero nunc. Ultricies integer quis auctor elit sed vulputate mi
+      sit amet. Egestas dui id ornare arcu odio ut. In iaculis nunc sed augue. Pellentesque
+      adipiscing commodo elit at imperdiet dui accumsan sit amet. Erat velit scelerisque in dictum
+      non. Auctor augue mauris augue neque gravida in fermentum et. Posuere sollicitudin aliquam
+      ultrices sagittis orci a scelerisque purus. Ullamcorper dignissim cras tincidunt lobortis
+      feugiat vivamus at augue. Tincidunt vitae semper quis lectus nulla. Purus ut faucibus pulvinar
+      elementum integer enim neque volutpat. Etiam sit amet nisl purus in mollis nunc. Diam sit amet
+      nisl suscipit. Nulla pharetra diam sit amet nisl. Arcu odio ut sem nulla.
+    </p>
+  );
+  const defaultBody = loremIpsum;
+  const defaultHeader = 'Drawer Header';
+  const defaultFooter = (
+    <>
+      <Button variant="primary">Confirm</Button>
+      <Button variant="tertiary">Cancel</Button>
```

Comment:
if you look at the screen shot you notice the buttons are touching. This is because React removes the space from the JSX. The work around is something like this is to add a space.
```suggestion
      <Button variant="primary">Confirm</Button>{' '}
      <Button variant="tertiary">Cancel</Button>
```
A more fancy solutions like a ButtonGroup wrapper, or a spacing class is not something Helix currently provides AFIK...Am I right about that @100stacks?

Reply:
Yes @nicko-winner, JSX stripping spacing is a know issue: https://helixdesignsystem.github.io/helix-ui/guides/react-compatibility/

We do have the Button Set class `.hxButtonSet` that many be a possible solution: https://helixdesignsystem.github.io/helix-ui/components/button/#button-set

<img width="554" alt="Screen Shot 2020-05-16 at 5 17 12 PM" src="https://user-images.githubusercontent.com/10120600/82131276-179a4c80-9799-11ea-89b5-b5b7307b0622.png">

Reply:
@100stacks I made use of `hxButtonSet` in my latest commit